client/src/components/Shows/ViewShow/Form.js

```diff
@@ -11,19 +11,11 @@ import {
   createInstanceAndEditShow,
   updateShow,
 } from '../../../redux/show';
+import { getShowType } from '../../../utils/shows';
```

Comment:
@jberns This PR moves `getShowType` into a `utils/shows.js` file - does that work for you?